.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: node_js
 node_js:
-  - "12.18.4"
+- 12.18.4
 before_install: npm i -g rimraf
 before_deploy:
-  - npm run build
-  - npm run cp
-  - cd dist
+- npm run build
+- npm run cp
+- cd dist
 deploy:
   provider: npm
   email: eimi.okuno@bbc.co.uk
```

Comment:
This email is still mine, does it work without it??

Reply:
shoot, good shout